src/http/helpers/body-parser.js

```diff
@@ -38,6 +40,10 @@ module.exports = function parseBody (req) {
       }
     }
 
+    if (isPlainText || isXml) {
+      request.body = new Buffer.from(request.body, 'base64').toString()
```

Comment:
This one is subtly tricky. On its face I think this is a great change (and now is the right moment to do it). However, it is unclear to me what the user expectations are here.

Our current behavior is: if we can parse the body into an object (e.g. JSON or `form-urlencoded`), we do that, otherwise we leave the body as a base64 encoded string (which comes along with the `isBase64Encoded: true` property).

This behavior breaks norm that by providing an unencoded string as the body. Again, I think it's for the right reasons, and removes a minor nuisance, but may also not be what folks are expecting to happen. (Or may be EXACTLY what folks are expecting to happen.)

One thing we recently added was the `rawBody` property; this ensures that whether we parse the body or not, users always have access to the raw data. An approach we can take here would be to tell users that in Functions v5+, they should only consider `isBase64Encoded: true` to correspond to `rawBody`, and that `body` will always be parsed (if we can indeed parse it, which in this case we can).

I'm a bit torn between these approaches: a) keep our behavior consistent, even if it creates a minor inconvenience, or b) eliminate a minor inconvenience and introduce some new behavior.

I can get behind either approach, curious to hear what @brianleroux thinks!